src/Shared/E2ETesting/E2ETesting.props

```diff
@@ -27,29 +27,19 @@
     <Compile Include="$(SharedSourceRoot)E2ETesting\**\*.cs" LinkBase="Selenium" />
   </ItemGroup>
 
-  <ItemGroup>
```

Comment:
Nit: maybe we can create a new props file for Playwright tests and include these configs there instead of having it in each test project.

Reply:
when we decide we need it we can put the files in a common place and just link them on the csproj.